utils/ad_manager_utils.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * @public
+ * @class AdManagerUtils
+ * @classdesc Provides utility/helper functions for ad managers.
+ */
+var AdManagerUtils = function()
+{
+
+  /**
+   * Helper function to convert the HMS timestamp into milliseconds.
+   * @private
+   * @method AdManagerUtils#_convertTimeStampToMilliseconds
+   * @param {string} timeString The timestamp string (format: hh:mm:ss / hh:mm:ss.mmm)
+   * @returns {number||null} The number of milliseconds the timestamp represents. Returns null
+   * if an error occurs.
+   */
+  this.convertTimeStampToMilliseconds = function(timeString)
+  {
+    var milliseconds = null;
+    if (_isValidTimeString(timeString))
+    {
+      var hms = timeString.split(":");
+      // + unary operator converts string to number
+      var hoursInSeconds = +hms[0] * 60 * 60;
+      var minutesInSeconds = +hms[1] * 60;
+      var seconds = +hms[2];
+      milliseconds = (hoursInSeconds + minutesInSeconds + seconds) * 1000;
+    }
+    return milliseconds;
+  };
+
+  /**
+   * Helper function to convert a percentage representing time into milliseconds.
+   * @private
+   * @method AdManagerUtils#_convertPercentToMilliseconds
+   * @param {string} timeString The string that represents a percentage (format: [0, 100]%)
+   * @param {number} movieDuration The duration of the video - represented in seconds
```

Comment:
Can we rename this parameter? The util as defined shouldn't have anything to do with movies or their durations. Something like totalDuration might work. We should also update the description of the jsdoc to include a bit about how this duration value is used.